cmake/SetupCompilerOptions.cmake

```diff
@@ -338,16 +338,21 @@ blt_append_custom_compiler_flag(
 #
 
 if ( COMPILER_FAMILY_IS_MSVC AND NOT BUILD_SHARED_LIBS )
-  foreach(_lang C CXX)
-    foreach(_build
-            FLAGS FLAGS_DEBUG FLAGS_RELEASE
-            FLAGS_MINSIZEREL FLAGS_RELWITHDEBINFO)
-        set(_flag CMAKE_${_lang}_${_build})
-        if(${_flag} MATCHES "/MD")
-            string(REGEX REPLACE "/MD" "/MT" ${_flag} "${${_flag}}")
-        endif()
+  if ( ENABLE_MSVC_STATIC_MD_TO_MT )
+    foreach(_lang C CXX)
+      foreach(_build
+              FLAGS FLAGS_DEBUG FLAGS_RELEASE
+              FLAGS_MINSIZEREL FLAGS_RELWITHDEBINFO)
+          set(_flag CMAKE_${_lang}_${_build})
+          if(${_flag} MATCHES "/MD")
+              string(REGEX REPLACE "/MD" "/MT" ${_flag} "${${_flag}}")
+          endif()
+      endforeach()
     endforeach()
-  endforeach()
+  elseif (ENABLE_GTEST)
+    message(FATAL_ERROR
+      "For static linking with MS Visual Studio using GTEST, you must enable changing /MD to /MT.")
```

Comment:
Reference the option they turned off to cause this

Reply:
done